docs/tutorials/music.rst

```diff
@@ -112,26 +97,12 @@ how to create a Police siren effect::
             music.pitch(freq, 6)
 
 
-Notice how the ``music.pitch`` *method* is used in this instance. It expects a
-frequency. For example, the frequency of ``440`` is the same as a concert ``A``
-used to tune a symphony orchestra.
-
-In the example above the ``range`` function is used to generate ranges of
-numeric values. These numbers are used to define the pitch of the tone. The
-three arguments for the ``range`` function are the start value, end value and
-step size. Therefore, the first use of ``range`` is saying, in English, "create
-a range of numbers between 880 and 1760 in steps of 16". The second use of
-``range`` is saying, "create a range of values between 1760 and 880 in steps of
--16". This is how we get a range of frequencies that go up and down in pitch
-like a siren.
-
-Because the siren should last forever it's wrapped in an infinite ``while``
-loop.
-
-Importantly, we have introduced a new sort of a loop inside the ``while``
-loop: the ``for`` loop. In English it's like saying, "for each item in some
-collection, do some activity with it". Specifically in the example above, it's
-saying, "for each frequency in the specified range of frequencies, play the
-pitch of that frequency for 6 milliseconds". Notice how the thing to do for
-each item in a for loop is indented (as discussed earlier) so Python knows
-exactly which code to run to handle the individual items.
+Zwróć uwagę na to jak polecenie ``music.pitch`` jest w tym przypadku użyte.
+Wymaga podania częstotliwości. Np. częstotliwość o wartości ``440`` jest podobna do częstotliwości jakiegoś koncertu, natomiast ``A`` to częstotliwość odpowiadająca orkiestrze symfonicznej.
```

Comment:
Wymaga ona podania

Reply:
Na przykład częstotliwość ``440`` to koncertowe ``A`` używane do strojenia instrumentów w orkiestrze symfonicznej.

(Bardzo proszę żeby nie zmyślać zdań kiedy nie wiemy co znaczy oryginał.)